File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        nums = new Set(nums)
        let longest = 0
        for (let num of nums){
            if (!nums.has(num-1)){
                let l = 1
                while (nums.has(num+l)){
                    l++
                }
                longest = Math.max(longest,l)

            }
        }
        return longest
    }
}

File: Data Structures & Algorithms/majority-element/submission-2.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        d = defaultdict(int)
        mx = 0
        output = 0
        for n in nums:
            d[n]+=1
            if d[n]>=len(nums)//2:
                if d[n]>=mx:
                    mx = d[n]
                    output = n
        return output